shared/src/types/types.ts

```diff
@@ -27,16 +27,31 @@ export interface Violation {
   effort?: number;
 }
 
-export type ViolationWithID = Violation & {
+export type EnhancedViolation = Violation & {
   id: string;
+  rulesetName?: string;
+  violationName?: string;
 };
```

Comment:
I guess this is because we need it already...why not put the extra bits in there?